docs/taints-tolerations.md

```diff
@@ -0,0 +1,92 @@
+# Taints and Tolerations Workload
+
+The Taints and Tolerations workload playbook is `workloads/taints-tolerations.yml` and will run the Taints and Tolerations  workload on your cluster.
+
+The Taints and Tolerations workload's purpose is to validate if the OpenShift cluster can deploy 130 hello-pods with memory and CPU requests per tainted worker node.
+
+An OCP with 3 masters and 3 worker nodes is required.
```

Comment:
@wabouhamad better to mention the min worker instance type required to run this workload in the docs since it uses cpu and memory requests. Thoughts?